lightyear/src/utils/wrapping_id.rs

```diff
@@ -182,6 +152,11 @@ pub(crate) use wrapping_id;
 /// assert_eq!(wrapping_diff(0, 32768), -32768);
 /// ```
 pub fn wrapping_diff(a: u16, b: u16) -> i16 {
+    b.wrapping_sub(a) as i16
+}
+
+// TODO: Remove
```

Comment:
old impl needs to be removed as expressed in description

Reply:
Done